hue/hue/base/hue-mysql-dc.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
```

Comment:
Same question - should we use `Deployment`?

Reply:
Same answer. :smile: you tell me. :slightly_smiling_face: 

I have no preference. `Deployment` makes it more close to upstream kubernetes and lowers complexity of this resource manifest.

Reply:
Changed to `Deployment`